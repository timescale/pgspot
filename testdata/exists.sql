-- test if not exists

CREATE COLLATION IF NOT EXISTS collation3 (locale = 'fr_FR.utf8');
CREATE FOREIGN TABLE IF NOT EXISTS unsafe_table4(c4444 text) SERVER server4;
CREATE INDEX IF NOT EXISTS index5 ON table5(column5);
CREATE MATERIALIZED VIEW IF NOT EXISTS matview6 AS SELECT pg_catalog.now();
CREATE SCHEMA IF NOT EXISTS schema7;
CREATE SEQUENCE IF NOT EXISTS sequence8;
CREATE SERVER IF NOT EXISTS server9 FOREIGN DATA WRAPPER postgres_fdw OPTIONS (host 'foo', dbname 'foodb', port '5432');
CREATE TABLE IF NOT EXISTS table10(c10 text);
CREATE TABLE IF NOT EXISTS table11 AS SELECT pg_catalog.now();

