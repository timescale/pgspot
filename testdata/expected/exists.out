PS017: Unqualified object reference: collation3 at line 1
PS007: Unsafe object creation: collation3 at line 1
PS012: Unsafe table creation: unsafe_table4 at line 4
PS017: Unqualified object reference: unsafe_table4 at line 4
PS014: Unsafe index creation: index5 at line 5
PS017: Unqualified object reference: table5 at line 5
PS007: Unsafe object creation: matview6 at line 6
PS017: Unqualified object reference: matview6 at line 6
PS010: Unsafe schema creation: schema7 at line 7
PS011: Unsafe sequence creation: sequence8 at line 8
PS017: Unqualified object reference: sequence8 at line 8
PS013: Unsafe foreign server creation: server9 at line 9
PS012: Unsafe table creation: table10 at line 10
PS017: Unqualified object reference: table10 at line 10
PS007: Unsafe object creation: table11 at line 11
PS017: Unqualified object reference: table11 at line 11

 Errors: 9 Warnings: 7 Unknown: 0 

